src/components/Map/SearchPane/SearchBar/AutoSuggestEntities/AutoSuggestEntities.tsx

```diff
@@ -113,7 +113,7 @@ const AutoSuggestEntities: React.FC<PageProps> = (props) => {
     const handleInputFocus = async() => {
       props.setIsSearching(true);
 
-      const response = await fetch('https://raw.githubusercontent.com/aldjonz/ccm-json/main/entities.json');
+      const response = await fetch(`${process.env.REACT_APP_DATABASE_URL}aldjonz/ccm-json/main/entities.json`);
       const data = await response.json();
```

Comment:
I suggest leaving the trailing slash off the .env URL like you have for `REACT_APP_BASE_URL` - it reads a bit better . Also I think the `REACT_APP_DATABASE_URL` should include `/aldjonz/ccm-json/main` - this isn't going to be the path on the live version